packages/cli/test/mocks/matchers/index.ts

```diff
@@ -11,4 +11,5 @@ declare module 'vitest' {
       ToHaveTelemetryEventsMatchers<T> {}
 }
 
+// @ts-expect-error
 expect.extend({ toOutput, toHaveTelemetryEvents });
```

Comment:
Couldn't find a great TS solution here, but I think`@ts-expect-error` is fine here because of the
```ts
declare module 'vitest' {
  interface Assertion<T = any>
    extends ToOutputMatchers<T>,
      ToHaveTelemetryEventsMatchers<T> {}
}
```
above.

Reply:
If we leave in a `@ts-expect-error`, let's also leave a comment about why

Reply:
Figured this out so no more `@ts-expect-error` is needed! `@vitest/expect`'s version was out of date with `vitest`, so it provided incompatible types.